.github/workflows/ci.yml

```diff
@@ -4,9 +4,14 @@ on:
   push:
     branches: [master]
   pull_request:
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
```

Comment:
This'll corrupt the raw output, like:
```
2023-02-09T12:35:09.8812217Z [0m[0m[1m[32m    Updating[0m crates.io index
2023-02-09T12:36:01.6195966Z [0m[0m[1m[32m Downloading[0m crates ...
2023-02-09T12:36:01.8229703Z [0m[0m[1m[32m  Downloaded[0m clap v2.34.0
2023-02-09T12:36:01.8339761Z [0m[0m[1m[32m  Downloaded[0m rand_core v0.6.4
2023-02-09T12:36:01.8360943Z [0m[0m[1m[32m  Downloaded[0m rayon-core v1.10.2
2023-02-09T12:36:01.8413414Z [0m[0m[1m[32m  Downloaded[0m regex-automata v0.1.10
2023-02-09T12:36:01.8500881Z [0m[0m[1m[32m  Downloaded[0m serde_cbor v0.11.2
2023-02-09T12:36:01.8541017Z [0m[0m[1m[32m  Downloaded[0m serde_json v1.0.93
```

Reply:
but at the same time it improves the readability of errors in CI
![image](https://user-images.githubusercontent.com/109800286/217832672-747c962e-526c-4742-a8ef-bd89da02717b.png)

Reply:
It's a trade-off, and I don't see any reason to do it here. It should be deferred to the maintainers' preference.